### PR TITLE
express-postgres - Skylar Shaffer

### PR DESCRIPTION
**Pull request by skylarshaffer**

### CODE REVIEW

File: express-postgres/server.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    const sql = `
+      select
+        "filmId",
+        "title",
+        "releaseYear"
+      from "films";
+    `;
+    const result = await db.query(sql);
+    const films = result.rows;
+    if (!films) {
```
Comment (RobertGardner) [inline express-postgres/server.ts:30]: It's not horrible to return a 404 if there are no films, but it's more common to just return an empty array.

File: express-postgres/server.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    next(err);
+  }
+});
+
+//  PUT /api/film updates a single film title, based on filmId. Pass filmId and title as query parameters. Return a success message. Return a 404 Not Found if the filmId is not in the database.
+
+app.put('/api/film', async (req, res, next) => {
+  try {
+    const { filmId, title } = req.query;
+    const missing = [] as string[];
```
Comment (RobertGardner) [inline express-postgres/server.ts:72]: I'm guessing TypeScript can figure out this is a string[]. Give it a try.

**Review by RobertGardner — APPROVED** [review]

File: express-postgres/server.ts (hunk 1 above)
Comment (skylarshaffer) [reply express-postgres/server.ts]: Thanks. Fixed.

File: express-postgres/server.ts (hunk 2 above)
Comment (skylarshaffer) [reply express-postgres/server.ts]: Thanks. Fixed.

**Pull request merged**